tests/integration/delete/delete_integ_base.py

```diff
@@ -38,5 +46,9 @@ def get_delete_command_list(
             command_list += ["--profile", str(profile)]
         if no_prompts:
             command_list += ["--no-prompts"]
+        if s3_bucket:
+            command_list += ["--s3-bucket", str(s3_bucket)]
```

Comment:
We can set the type information of `s3_bucket` parameter to `Optional[str]` then we don't need to wrap it with `str` call here.